dist/profile/templates/kubernetes/resources/evergreen_gateway/deployment.yaml.erb

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: gateway
+  namespace: evergreen
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: gateway
+    spec:
+      containers:
+        - name: gateway
+          image: jenkinsciinfra/sshd:evergreen-<%= @parameters['image_tag'] %>
```

Comment:
I believe this should be `jenkinsciinfra/sshd:<%= @parameters['image_tag'] %>`

Based on how `$image_tag` is passed in currently, this will result in the image tag being: `jenkinsciinfra/sshd:evergreen-evergreen-d69c65`

(redundant `evergreen` components)

Reply:
Thanks nice catch